tutorials/governance/opengov/create-a-referenda.md

```diff
@@ -0,0 +1,91 @@
+4. After submitting, you should see a `referenda.Submitted` event with your proposal in the explorer section. The proposal number will be displayed in the **index** field
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-7.webp)
+
+## Submit a Referendum on the Whitelisted Caller Track
+
+Operations deemed safe or time-sensitive by the Polkadot Technical Fellowship can be processed through the Whitelisted Caller track. This specialized track allows proposals to pass more quickly by requiring less voter turnout during the early stages of the decision period. It is primarily used for technical, neutral proposals, such as runtime upgrades or adjustments to the system’s parachain validation configuration. However, unlike other tracks, the Whitelisted Caller track requires a different approach. Instead of voting directly on the proposal, the vote is to dispatch the proposal via the Whitelist pallet.
+
+Before submitting a referendum on this track, it’s crucial to secure a positive signal from the Fellowship that they will whitelist the proposal. Without this endorsement, it won’t be executed even if the referendum passes.
+
+For instance, if you want to increase the number of validators in the parachain consensus, you could submit a preimage with a call to set the number of validators to 1,000 and propose it directly through the Root track. However, this would require a significant decision deposit and face conservative passing criteria, likely needing the entire 28-day voting period to succeed.
+
+To submit a proposal on the Whitelisted Caller track, follow these steps:
+
+1. Construct the extrinsic you want to execute and submit the preimage:
+    1. Select the **whitelist** pallet
+    2. Choose the **dispatchWhitelistedCallWithPreimage** extrinsic
+    3. Build the `call` you want to execute and fill the parameters
+    4. Copy the preimage hash -
+    the preimage hash for this example is `0xb2dc8750d1736510a329cf9f7e74cd58376d451d33109807a2909ae30e8a72cf`
+    5. Click the **+ Submit preimage** button
+    ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-10.webp)
+
+2. Follow the steps in section [Submit a Proposal](#submit-a-proposal) to submit your proposal using the preimage hash you obtained in the previous step. Ensure that you select the `Whitelisted Caller` track
+   
+For this proposal to succeed, the Polkadot Fellowship must initiate a Fellowship referendum to whitelist the call using the `whitelist.whitelistCall(callHash)` extrinsic.
+
+To obtain the  call hash for the `call` you want to execute:
+
+1. Navigate to the **Developer** dropdown and select the **Extrinsics** option
+  ![](/images/tutorials/governance/opengov/create-a-referenda/create-a-referenda-8.webp)
+2. Choose the appropriate pallet and extrinsic, fill in the required parameters for the call, and copy the **encoded call hash**
```

Comment:
it might be helpful to include a note below the image explaining the pallet and extrinsic used for this specific example, like:

!!! note
    This example uses **configuration** pallet and **setMax** extrinsic to...